src/lib/loaders/loaders_without_authentication/gravity.js

```diff
@@ -29,19 +29,41 @@ export default requestIDs => {
       {},
       { headers: true }
     ),
+    partnerArtistLoader: gravityLoader(
+      ({ artist_id, partner_id }) => `partner/${partner_id}/artist/${artist_id}`
```

Comment:
We have a few loaders that have to take an object, rather than just an `id`, works 👍